settings.gradle

```diff
@@ -18,6 +18,7 @@ List projects = [
   'test:fixtures:example-fixture',
   'test:fixtures:hdfs-fixture',
   'test:logger-usage',
```

Comment:
If you are going to make a `:test:tools` then we should move logger-usage into it eventually. Probably in another PR.